samples/OcelotServiceDiscovery/ApiGateway/appsettings.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  "Logging": {
+    "IncludeScopes": true,
+    "LogLevel": {
+      "Default": "Information",
+      "System": "Information",
+      "Microsoft": "Information"
+    }
+  },
+  "spring": {
+    "application": { "name": "Ocelot-Gateway" },
+    "cloud": {
+      "config": {
+        "uri": "http://localhost:5000",
+        "validateCertificates": false
+      }
+    }
+  }
```

Comment:
```json
  "spring": {
  }
```
What is this config section about?
What tool/software is used here?

Reply:
The spring config sections in both appsettings.json files seem to be a leftover from the Eureka project this sample was based on. They should be able to be safely removed without issue.

Reply:
Please, remove this redundant section!